packages/remix/src/utils/versions.ts

```diff
@@ -27,3 +29,11 @@ export function getPackageVersion(tree: Tree, packageName: string) {
     null
   );
 }
+
+/**
+ * Infer bundler type depending on vite.config.ts presence
+ * @param root workspace root path
+ */
+export function getBunlderType(root: string): 'classic' | 'vite' {
+  return existsSync(join(root, 'vite.config.ts')) ? 'vite' : 'classic';
```

Comment:
It would be safer to load the config file and check that remix is a plugin: https://github.com/nrwl/nx/pull/22551/files#diff-e21b84bc7a86a657a55ca6be0f7f8dbe1c2a3ff3475357e0de168fc775c987bbR130